Port lang

TehPeGaSuS opened the pull request:
Found by:
Patch by: TheMythPT
Fixes: 

One-line summary:


Additional description (if needed): Finished translating core messages into Portuguese language.


Test cases demonstrating functionality (if applicable):

Commit:
adding portugueselang (#317)

TehPeGaSuS committed:
Update portugese language file

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,438 @@
+0x668,A partir de agora, não precisas de usar a opção -m quando iniciares o bot.\nDesfruta!!
+0x669,Flood de ligação telnet de %s! Colocando em ignore!
+0x66a,Banido:
+0x66b,join flood
```
Needs translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,438 @@
+0x670,Removida excepção
+0x671,Removido convite
+0x672,NICK flood de @%s! Banindo.
+0x673,nick flood
```
Needs translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,438 @@
+0x909,Abuso de op mal atribuido pelo servidor
+0x90a,Alteração de modo por um não op no %s!  Revertendo...
+0x90b,Abuso de desync
+0x90c,flood
```
Needs translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,438 @@
+0xb26,Impossível ligar lá
+0xb27,Impossível desligar
+0xb28,Loop detectado
+0xb29,Bogus link notice from
```
Needs translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,438 @@
+0xb5c,Terminando ligação relay para
+0xb5d,Interrompendo ligação para
+0xb5e,Relay interrompido
+0xb5f,Ping timeout
```
Needs translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,438 @@
+0xc01,Recusado DCC chat (sem acesso)
+0xc02,Sem acesso
+0xc03,Deves ter uma password definida.
+0xc04,Recusado DCC chat (sem password)
```
"password" needs translation (and many more through the file)?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,438 @@
+0xe04,entrou no
+0xe05,está agora ausente
+0xe06,não está mais ausente
+0xe07,Mudança de nick:
```
"nick" needs translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,438 @@
+0xe2f,Tempo limite de conexão/EOF esgotado
+0xe30,Perdida socket telnet aguardando por ident!!
+0xe31,Ligação telnet recusada: %s,Sem Acesso
+0xe32,Ligação telnet recusada de %s (tentou usar o meu nick de botnet)￿
```
Missing 0xe33

Cizzle committed:
Correct spacing, casing and add new strings.

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,439 @@
+0x40c,ERRO na escrita de arquivo de usuário.
+0x40d,ERRO escrevendo arquivo de usuário para transferência.
+0x40e,Criação de arquivo de usuário não é necessária--ignorando
+0x40f,Rehashing...
```
Needs translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,439 @@
+0x50d,es
+0x50e,Mudando logfiles...
+0x512,inativo
+0x513,AWAY
```
Needs translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,439 @@
+0x602,Queria me banir--desavisado.
+0x603,foi divertido, vamos fazer de novo!
+0x604,Hi
+0x605,Goodbye
```
Needs translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,439 @@
+0x63c,O servidor diz que ainda não estamos registados...
+0x63d,Flood de @%s! Colocando em ignorar!
+0x63f,JOIN inundação de @%s! Banning.
+0x640,Inundação de canal de %s -- kicking
```
"Banning" and "kicking" need translation?

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,439 @@
+0x648,Fila do servidor está em
+0x649,Fila de ajuda está em
+0x64a,Vhost set in config can not be used to connect to this address
+0x64b,Vhost set in config is not available on this machine
```
2 new strings needing translation.

Cizzle commented on language/core.portuguese.lang:
```diff
@@ -0,0 +1,439 @@
+0xe30,Perdida socket telnet aguardando por ident!!
+0xe31,Ligação telnet recusada: %s, Sem Acesso
+0xe32,Ligação telnet recusada de %s (tentou usar o meu nick de botnet)
+0xe33,Lost telnet connection from %s while checking for duplicate
```
Missing string needs translation.

vanosg commented:
@TheMythPT Are you still active? Could you stop in on freenode?

TehPeGaSuS committed:
Update core.portuguese.lang

TehPeGaSuS committed:
Update core.portuguese.lang

Fixed some typos.
This time i guess it can be marked as official!
Enjoy!

TehPeGaSuS committed:
Update core.portuguese.lang

Fixed translation typo, at 0x405, where quotes were removed from the command explanation (found by Cizzle)

TehPeGaSuS committed:
Update core.portuguese.lang

Fixed all the remaining typos.
Translated background word (even if i dont like the portuguese term., makes more sense to newbies), pointed by Cizzle.

TehPeGaSuS committed:
Update core.portuguese.lang

Reviewed once again.
Removed 0x700 block.
Translated unused.

Cizzle committed:
Some final fixes.
